modules/javafx.graphics/src/test/java/test/javafx/scene/CssStyleHelperTest.java

```diff
@@ -60,7 +70,7 @@ private static void resetStyleManager() {
         sm.hasDefaultUserAgentStylesheet = false;
     }
 
-    @Before
+    @BeforeEach
```

Comment:
since we are now creating Stage every time, shouldn't we `hide()` it each time?

Reply:
This was already done each time, it has not changed, only the annotation.  The semantics of `@Before` in JUnit 4 and `@BeforeEach` in JUnit 5 are the same.  However, see #1515 as I'll remove these changes from this PR.

Reply:
I'm not entirely sure on the semantics of `Stage` when it is run with the stub toolkit; I suspect they won't stick around when no longer referenced, or this test would have been an issue when it was created years ago already.

Reply:
The annotation changes have now been removed from this PR now that #1515 has been integrated.

Reply:
This question may not relate to this PR as it does not change the existing logic.

I wonder if the lingering Stages affect the test(s) in some way.  Is FX subsystem getting shut down after each test class?  Does anyone know?